packages/datagateway-dataview/cypress/integration/table/dls/datasets.spec.ts

```diff
@@ -264,6 +264,25 @@ describe('DLS - Datasets Table', () => {
       cy.contains('4.55 GB', { timeout: 10000 }).should('be.visible');
     });
 
+    it('and then calculate file size when the value is 0 ', () => {
+      // need to wait for counts to finish, otherwise cypress might interact with the details panel
+      // too quickly and it rerenders during the test
```

Comment:
I think this comment should be moved one line below as it refers to the .should('exist').